elements-fun/src/transaction.rs

```diff
@@ -170,14 +332,14 @@ impl TxInWitness {
 }
 
 /// Parsed data from a transaction input's pegin witness
-#[derive(Copy, Clone, Default, PartialEq, Eq, Debug, Hash)]
+#[derive(Copy, Clone, PartialEq, Eq, Debug, Hash)]
 pub struct PeginData<'tx> {
     /// Reference to the pegin output on the mainchain
     pub outpoint: bitcoin::OutPoint,
     /// The value, in satoshis, of the pegin
     pub value: u64,
     /// Asset type being pegged in
-    pub asset: confidential::Asset,
+    pub asset: AssetId,
```

Comment:
:question: Does this mean that this struct is always explicit?

Reply:
Yes from my understanding yes!